src/modules/private-share-folder/private-sharing.controller.ts

```diff
@@ -239,4 +245,65 @@ export class PrivateSharingController {
         ),
     };
   }
+
+  @Delete('stop/folder-id/:folderId')
+  @ApiOperation({
+    summary: 'Stop sharing one folder',
+  })
+  @ApiBearerAuth()
+  async stopSharing(
+    @Param('folderId', ParseUUIDPipe) folderUuid: string,
+    @UserDecorator() user: User,
+  ): Promise<any> {
+    try {
+      return await this.privateSharingUseCase.stopSharing(folderUuid);
+    } catch (error) {
+      if (
+        error instanceof FolderNotSharedError ||
+        error instanceof NotFoundException
+      ) {
+        throw error;
+      }
+      new Logger().error(
+        `[PRIVATESHARING/STOP] ERROR: ${
+          (error as Error).message
+        }, Error while stopping shared folder by folder ${folderUuid}, ${
+          error.stack || 'No stack trace'
+        }`,
+      );
+    }
+  }
+
+  @Delete('remove/folder-id/:folderId/user-id/:userId')
```

Comment:
The HTTP Method is DELETE, `remove` word usage then, is redundant. Also, matching with the private-sharing id as the above endpoint is enough, I think